Wessel/FileChooser.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+using UnityEngine.UI;
+
```

Comment:
Ik vind dit prima code. Ben het niet eens met dat élke public var een property moet hebben. Dat maakt de code 2x onpraktischer, voller en maakt weinig verschil.

Je had de haakjesregel bij line 26 al gefixt, maar de nieuwste commit is niet de hele file, dus daarom zeg ik dit even hier.

Good going!